website/docs/r/hosting_privatedatabase_database.html.markdown

```diff
@@ -0,0 +1,35 @@
+---
+layout: "ovh"
+page_title: "OVH: ovh_hosting_privatedatabase_database"
+sidebar_current: "docs-ovh-resource-hosting-privatedatabase-database"
+description: |-
+  Create a new database on your private cloud database service.
+---
+
+# ovh_hosting_privatedatabase_database
+
+Create a new database on your private cloud database service.
+
+## Example Usage
+
+```hcl
+resource "ovh_hosting_privatedatabase_database" "database" {
+  service_name  = "XXXXXX"
+  database_name = "XXXXXX"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `service_name` - The internal name of your private database.
+* `database_name` - (Required) Name of your new database
+
+## Attributes Reference
+
+The id is set to the value of `service_name`/`database_name`.
+
+## Import
```

Comment:
Please follow the same import block like other resources.
Example:

```

## Import

OVHcloud Managed Kubernetes Service clusters can be imported using the `serviceName` and the `id` of the cluster, separated by "/" E.g.,

```bash
$ terraform import ovh_cloud_project_kube.my_kube_cluster service_name/kube_id
```